torchvision/transforms/functional_pil.py

```diff
@@ -154,7 +154,7 @@ def pad(
 
     if not isinstance(padding, (numbers.Number, tuple, list)):
         raise TypeError("Got inappropriate padding arg")
-    if not isinstance(fill, (numbers.Number, str, tuple)):
+    if not isinstance(fill, (numbers.Number, str, list, tuple)):
```

Comment:
Unrelated bug fix on the original code.

Reply:
If there is a bug fix here and we do not expect this to land, maybe better to split this into a separate PR ?

Reply:
Yeah we can cherrypick afterwards if we don't land this.